lib/FileSystemInfo.js

```diff
@@ -78,6 +88,29 @@ const mergeMaps = (a, b) => {
 	return map;
 };
 
+/**
+ * Finding file or directory to manage
+ * @param {string} managedPath path that is managing by {@link FileSystemInfo}
+ * @param {string} path path to file or directory
+ * @returns {string|null} managed item
+ * @example
+ * getManagedItem(
+ *   '/Users/user/my-project/',
```

Comment:
managedPath is always a path ending with `node_modules/`. The examples should represent that.